private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -410,6 +405,7 @@ export class TemplateServiceClient {
           templateInstance.state = templateInstance.state !== TemplateState.deprecated && templateInstance.state !== TemplateState.draft &&
             instance.state !== TemplateState.draft ? TemplateState.draft :
             templateInstance.state || instance.state;
+          templateInstance.author = instance.author; // author should not be able to be overwritten
```

Comment:
If a template is published and someone who is not the author of that version of the template unpublished it, then will the original author be maintained?

Reply:
yes. that is the whole idea.

I think if a version goes through changes, the original author should remain the same through all changes in that specific version.

Reply:
My one concern is that other people can change the template version. If they do change the version, it's as if they're making changes under the author's name, instead of their own name. 

This may be simply a limitation of constructing the owner system this way. I think in an ideal world, we would be keeping track of all people that have changed the template. Something like what GitHub does: 
![image](https://user-images.githubusercontent.com/40618774/78699184-5561a680-78b8-11ea-8809-9fa961fcd2d8.png)
But, I'm sure that's out of scope of our internship :(

Reply:
i agree, it can be a little misleading. however, the `author` is not actually displayed in `templateinfo`, rather we show the `lastediteduser`. author is not really a visible field so i think its not a concern.

Reply:
The designs are adjusted to show `Last Edited By` rather than `author`